src/datamapper/AusenciaJpaController.java

```diff
@@ -170,9 +173,9 @@ public Ausencia findAusencia(Long id) {
     public Ausencia findAusencia(String codigo){
     List<Ausencia> ausencias = this.findAusenciaEntities();
     for(Ausencia ausencia : ausencias){
-        if(ausencia.getCodigo().equals(codigo))
+        if(ausencia.getCodigo().equals(codigo)){
             return ausencia;
-    }
+    }}

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['for rcurly' have incorrect indentation level 5, expected level should be 8.](https://www.codacy.com/app/calebepb/substituicao/file/2738539596/issues/source?bid=3308597&fileBranchId=3353540#l178)